pkg/openevec/config.go

```diff
@@ -252,11 +252,7 @@ func Merge(dst, src reflect.Value, flags *pflag.FlagSet) {
 		if dst.Type().Field(i).Tag != "" {
 			tag := dst.Type().Field(i).Tag.Get("cobraflag")
 			if tag != "" && flags.Changed(tag) {
-				if dst.CanSet() {
```

Comment:
What are the test cases, where the field returns false for CanSet() function?

Reply:
The real change here is about usage of Field(i) instead of the whole object to override. I have no test cases for false from CanSet, so that was removed as you can see

Reply:
Actually I made additional change here: if no mapstructure tag define we are not able to load the item from config, so set value from flag (at least from its defaults) is better approach